tests/playwright/src/fixtures/chat-fixtures.ts

```
@@ -0,0 +1,87 @@
+  resourceSetup: void;
+}
+
+interface TestFixtures {
+  electronApp: ElectronApplication;
+  page: Page;
+}
+
+export const test = base.extend<TestFixtures, WorkerFixtures>({
+  resource: [
+    // eslint-disable-next-line no-empty-pattern, @typescript-eslint/no-explicit-any
+    async ({}, use): Promise<void> => {
```

Biome (2.1.2)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)